docs/03-policies/prodev.md

```diff
@@ -0,0 +1,102 @@
+
+The total time commitment for participating in a prodev sprint will vary widely depending on what topic and intensity level each participant chooses to pursue, but the prodev sprint is not considered a full time project.
+
+Assignment to a prodev sprint is voluntary, and team members are encouraged to express their desire to participate to the prodev Practice Lead, Jason Heaster. Efforts will be made to group people with similar prodev goals or certification requirements. Once assigned to a sprint, participants’ project assignment schedules will be adjusted in Forecast to allocate time for the prodev sprint during that month.
+
+## Making a Plan
+
+Previously, CivicActions had a professional development process that included regular management review of prodev plans created on trello boards. While this isn’t a required practice anymore, you are still encouraged to take responsibility for your own professional development by creating a prodev plan and setting goals for yourself. If you would like help with structuring a plan, use [this trello board template](https://trello.com/b/p7FOD0Ju/template-professional-development-and-community-participation) as a guide or ask the prodev Practice Lead for help.
+
+## Asking a Mentor, Coach, or Peer to Help
+
+For some people, getting input from others is an important part of the professional development process. You are encouraged to seek out other CivicActioners who you think could help you achieve your prodev goals. This may be a manager, a peer, or any person who is good at something you’d like to learn.
+
+Also, if you’re asked by someone else to assist with their prodev, please make time to help them out!
+
+## Communication
+
+We have a [#prodev](https://civicactions.slack.com/messages/prodev) Slack channel for prodev-related communication.
+
+## Prodev FAQ's
+
+Q: How much do I get to spend on prodev each year?  
+A: Your budget is $1,200. If you’re just starting, you’ll need to wait 90 days before you gain access to your prodev budget. If you’re a part-time employee, your budget may be less than $1,200. Your manager can tell you more.
+
+Q: What kinds of stuff can I spend my prodev budget on?  
+A: Books, courses, conferences, certifications and prodev-related travel expenses.
+
+Q: Are there any other limits on the prodev budget?  
+A: For spending amounts over $50, you’ll need to get approval in advance. [Use this form](https://goo.gl/forms/4z1GQsBtrx6m4lVy2) to request approval. Anything under $50 can be purchased without permission as long as it supports professional development goals. Receipts and expense sheets should be submitted in the usual way, noting as professional development costs.
```

Comment:
"the usual way" -- we should probably link to the handbook page that mentions Harvest (does this exist?)